simtools/data_model/metadata_collector.py

```diff
@@ -268,13 +268,18 @@ def _read_input_metadata_from_file(self, metadata_file_name=None, observatory="C
             self._logger.debug("No input metadata file defined.")
             return {}
 
-        # metadata from yml file
-        if Path(metadata_file_name).suffix == ".yml":
+        # metadata from yml or json file
+        if Path(metadata_file_name).suffix in (".yaml", ".yml", ".json"):
             try:
                 self._logger.debug("Reading meta data from %s", metadata_file_name)
                 _input_metadata = gen.collect_data_from_file_or_dict(
                     file_name=metadata_file_name, in_dict=None
                 )
+                _json_type_metadata = {"Metadata", "metadata", "METADATA"}.intersection(
+                    _input_metadata
+                )
+                if len(_json_type_metadata) > 0:
+                    _input_metadata = _input_metadata[_json_type_metadata.pop()]
```

Comment:
It is not totally clear to me the idea behind these lines here. If there are at least one "metadata" key in `_input_metadata` we only take the last one? Could you please explain me?

Reply:
We cannot expect at this point that the metadata keys are all lower case (as they will be later). So these four lines make sure that any key from `{"Metadata", "metadata", "METADATA"}` is accepted.

Reply:
Any key, but it always takes the last one (that is what `.pop()` does) if there are more than one key among `{"Metadata", "metadata", "METADATA"}`. My point is that I do not see the reason why we want to take the last one specifically. I think we should indeed accept any of the keys but if there are more than one, we should raise a warning or e.g. take the metadata from both keys, no?

Reply:
OK, this is a very unlikely event!

Anyway, I've added a check for this now (and a unit test).

Reply:
Indeed. Ok, looks good, thanks!